src/test/resources/expected_helm_output/jira/output.yaml

```diff
@@ -168,20 +168,13 @@ spec:
               mountPath: "/opt/atlassian/jira/atlassian-jira/WEB-INF/atlassian-bundled-plugins/myplugin.jar"
               subPath: "plugins/myplugin.jar"
       volumes:
+        - name: local-home
+          emptyDir: {}
         - name: shared-home
-          persistentVolumeClaim:
-            claimName: jira-shared-home
```

Comment:
Could we have another "unit test" that would assert what happens when `persistence.enabled=true`?

Reply:
Yes, we really should. I'm chewing on an idea for a better way of unit testing these helm templates, but for now we're relying on the deployment integration tests to test that.

Reply:
@kcichy-atlassian please see #23 for the "better way". I haven't added tests for this specifically yet, but I will.